keyring/ssh_ecdsa_256_test.go

```diff
@@ -0,0 +1,27 @@
+package keyring
+
+import (
+	"testing"
+)
+
+func TestSSHECDSA256(t *testing.T) {
```

Comment:
I think all of these SSH key test cases could just go into 1 file?

And missing a test for regular PEM encoded RSA/EC key.